README.md

```diff
@@ -320,8 +329,15 @@ Statics parameters
   - Default: `.build/css`
 - `jsPath`: Subdirectory within `staticsRoot` where your JS files are located. By default this folder will not be made public, but is instead meant to store unminified JS source files which will be minified and stored elsewhere when the app is started.
   - Default: `js`
+- `bundledJsPath`: Subdirectory within `staticsRoot` where you would like [browserify](http://browserify.org) to deposit bundled JS files it produces (if you use browserify).
+  - Default: `js/.bundled`
+- `browserifyBundles`: Declare [browserify](http://browserify.org) bundles: one or more files in your `jsPath` for [browserify](http://browserify.org) to bundle via its [bundle method](https://github.com/substack/node-browserify#browserifyfiles--opts).
+  - Default: `[]`
+  - Example declaring one bundle: `[{outputFile: "bundle.js", files: ["landingPage.js", "main.js", "etc.js"], params: {someOpt: "someValue"}}]`
+  - Example declaring multiple bundles: `[{outputFile: "bundle1.js", files: ["landingPage.js", "main.js", "etc.js"], params: {someOpt: "someValue"}}, {outputFile: "bundle2.js", files: ["somethingElse.js", "anotherThing.js", "etc.js"]}, etc...]`
```

Comment:
the whole README needs refactoring so the examples are valid JSON, which actually presents some interesting formatting challenges since we now have lots of examples that are multiline. i'll file a separate issue on that